src/serializers/xml/helpers.js

```diff
@@ -9,3 +9,62 @@ export function equals(leftVal, rightVal, options) {
 		return options.fn(this);
 	}
 }
+
+// Implementation from http://stackoverflow.com/a/2893259
+export function formatXml(xml) {
```

Comment:
It's a little funny that a handlebars helper and a non handlebars helper are in the same file. Not a big deal just has the potential to be misleading.